docs/streams/developer-guide/dsl-api.html

```diff
@@ -3134,6 +3192,49 @@ <h4><a id="streams_concepts_globalktable" href="#streams_concepts_globalktable">
                 </div>
             </div>
         </div>
+	
+	<div class="section" id="controlling-update-rate">
+            <span id="streams-developer-guide-dsl-suppression"></span><h2><a class="toc-backref" href="#id32">Controlling KTable update rate</a><a class="headerlink" href="#controlling-update-rate" title="Permalink to this headline"></a></h2>
```

Comment:
Not sure if this is technically correct? The `KTable` itself is updated for each input record. Suppress throttles downstream update rate though. (maybe too nitty?)

Reply:
or maybe "emit rate"?

Reply:
`emit rate` works for me

Reply:
Like the term "emit rate"